### DISCUSSION TITLE
Do you know how to solve this problem when make

### DISCUSSION THREAD

**Discussion by 123xyl** [openmopac/mopac#106] [Q&A]
![image](https://user-images.githubusercontent.com/96686347/184533180-3b0b14d4-b68e-4e1f-955b-1b0a68a9b2aa.png)

**Comment by godotalgorithm** [comment]
`iargc` is an intrinsic Fortran subroutine that any Fortran-77-compatible compiler should automatically recognize. Which Fortran compiler are you using, and what compiler flags are you using? I am not familiar with any Fortran compilers that do not support `iargc`, but there are strict compatibility modes such as`-std=f95` in gfortran that can cause failures like this.

**Reply by 123xyl** [reply]
Do you know where the iarc function is defined?

**Reply by godotalgorithm** [reply]
I have attempted to answer this in #107.
